gtsfm/utils/io.py

```diff
@@ -55,6 +55,7 @@ def load_image(img_path: str) -> Image:
         exif_data = parsed_data
 
     img_fname = Path(img_path).name
+    original_image = original_image.convert("RGB") if original_image.mode != "RGB" else original_image
```

Comment:
looks like something that should be documented in the function docstring.

Reply:
+1

Reply:
added